discovery-infra/test_infra/controllers/node_controllers/terraform_controller.py

```diff
@@ -43,18 +43,12 @@ def _get_random_name(cls):
     # TODO move all those to conftest and pass it as kwargs
     # TODO-2 Remove all parameters defaults after moving to new workflow and use config object instead
     def _terraform_params(self, **kwargs):
-        operators = kwargs.get("olm_operators", [])
         params = {
-            "libvirt_worker_memory": operators_utils.resource_param(
-                kwargs.get("worker_memory"), consts.OperatorResource.WORKER_MEMORY_KEY, operators),
-            "libvirt_master_memory": operators_utils.resource_param(
-                kwargs.get("master_memory", 16984), consts.OperatorResource.MASTER_MEMORY_KEY, operators),
```

Comment:
You have that value on `consts.resources`  
https://github.com/openshift/assisted-test-infra/blob/master/discovery-infra/test_infra/consts/resources.py

Reply:
OK